features/steps/workspace_steps.js

```diff
@@ -1,4 +1,9 @@
 const { Given, When, Then } = require("cucumber");
+const WorkspacePage = require("../page-objects/WorkspacePage");
+
+Given('the {actor} is on the {workspace} Dashboard', async function (actor, workspace) {
+  await new WorkspacePage(this.driver).enter(workspace);
```

Comment:
Here is where I would do `this.workspace = ...` to provide the workspace to steps later on.